cm_sdk/client.py

```diff
@@ -0,0 +1,60 @@
+import requests
+from cm_sdk.api import APIException
+
+class CloudManagerClient(object):
+    """
+    Simple client for interacting over HTTP with the the MongoDB
+    CloudManager API.
+    """
+
+    HEADERS = {'Content-type': 'application/json'}
+
+    def __init__(self, api_host, group_id, api_user, api_key):
+        self.base_url = api_host + "/api/public/v1.0"
+        self.group_id = group_id
+        self.http_digest_auth = requests.auth.HTTPDigestAuth(api_user, api_key)
+
+
+    def _get_final_url(self, url_template):
+        return self.base_url + url_template.format(group_id=self.group_id)
+
+
+    def _valid_response(self, response):
+        if 'error' in response:
+            raise APIException(response['error'])
+        return response
+
+
+    #
+    # HTTP Request methods
+    #
+    def get(self, path):
+        r = requests.get(self._get_final_url(path), auth=self.http_digest_auth)
+        return self._valid_response(r.json())
```

Comment:
The `r.json()` call could fail if the response is not valid json, do we want to catch that and wrap it in an `APIException`?